stories/components/peoplePicker/peoplePicker.properties.js

```diff
@@ -213,3 +213,32 @@ export const pickerGroupFilters = () => html`
     type="group">
   </mgt-people-picker>
 `;
+
+export const pickerGroupIds = () => html`
```

Comment:
This one doesn't seem to work. It displays the groups... But not the users withing these groups? (Finance and Marketing)

![image](https://user-images.githubusercontent.com/7620955/168895923-17b47a8a-140c-48af-879e-67d3f2ebf059.png)

Reply:
I suspect the store version is reading and loading stale results. Made a version bump on it, should work as expected.